examples/forward/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"golang.ngrok.com/ngrok/v2"
+)
+
+// example usage:
+// ./http https://mydomain.ngrok.app http://localhost:8080
```

Comment:
Did you mean `ngrok http --url=https://mydomain.ngrok.app http://localhost:8080` to show the equivalent cli command, or am I just not understanding this comment's syntax?

Reply:
it's actually the example way to call the executable created by this example. i suppose it should say `./forward ...` will fix and improve the comment to clarify